drracket/drracket/private/stick-figures.rkt

```diff
@@ -101,67 +101,63 @@
-             [bdc-solid (make-object bitmap-dc% bm-solid)]
-             [bdc-small (make-object bitmap-dc% bm-small)])
-        (send bdc-big clear)
-        (draw-callback bdc-big small-factor #f points
-                       (+ margin (- (* small-factor min-rx)))
-                       (+ margin #;(- (* small-factor min-ry)))
-                       3)
-        
-        (send bdc-small clear)
-        (send bdc-small set-scale small-bitmap-factor small-bitmap-factor)
-        (send bdc-small draw-bitmap bm-big 0 0)
-        (send bdc-small set-scale 1 1)
-        
-        (send bdc-solid set-brush green 'solid)
-        (send bdc-solid set-pen green 1 'solid)
-        (send bdc-solid draw-rectangle 0 0 w h)
-        
-        (send bdc-solid set-bitmap #f)
-        (send bdc-small set-bitmap #f)
-        (send bdc-big set-bitmap #f)
-        
-        (send bm-solid set-loaded-mask bm-small)
-        bm-solid)))
+  (define-values (min-rx min-ry) (get-max/min-x/y min points))
+  (define-values (max-rx max-ry) (get-max/min-x/y max points))
+  (define margin 2)
+  (define bw (+ margin margin (ceiling (* small-factor (- max-rx min-rx)))))
+  (define bh (+ margin margin (ceiling (* small-factor (- max-ry min-ry)))))
+  (define w (ceiling (* bw small-bitmap-factor)))
+  (define h (ceiling (* bh small-bitmap-factor)))
+  (define bm-big (make-object bitmap% bw bh))
+  (define bm-solid (make-object bitmap% w h))
+  (define bm-small (make-object bitmap% w h))
+  (define bdc-big (make-object bitmap-dc% bm-big))
+  (define bdc-solid (make-object bitmap-dc% bm-solid))
+  (define bdc-small (make-object bitmap-dc% bm-small))
+  (send bdc-big clear)
+  (draw-callback bdc-big small-factor #f points
+                 (+ margin (- (* small-factor min-rx)))
+                 (+ margin #;(- (* small-factor min-ry)))
+                 3)
+  (send bdc-small clear)
+  (send bdc-small set-scale small-bitmap-factor small-bitmap-factor)
+  (send bdc-small draw-bitmap bm-big 0 0)
+  (send bdc-small set-scale 1 1)
+  (send bdc-solid set-brush green 'solid)
+  (send bdc-solid set-pen green 1 'solid)
+  (send bdc-solid draw-rectangle 0 0 w h)
+  (send bdc-solid set-bitmap #f)
+  (send bdc-small set-bitmap #f)
+  (send bdc-big set-bitmap #f)
+  (send bm-solid set-loaded-mask bm-small)
+  bm-solid)
   
   (define (get-running-bitmap) (get-bitmap running-points (make-object color% 30 100 30)))
   (define (get-waiting-bitmap) (get-bitmap waiting-points (make-object color% 30 100 30)))
   
   (define (normalize points)
-    (let-values ([(min-x min-y) (get-max/min-x/y min points)])
-      (map (λ (x) (list (car x) 
-                        (+ (- (list-ref x 1) min-x))
-                        (+ (- (list-ref x 2) min-y))))
-           points)))
+  (define-values (min-x min-y) (get-max/min-x/y min points))
+  (map (λ (x) (list (car x) 
+                    (- (list-ref x 1) min-x)
+                    (- (list-ref x 2) min-y)))
+       points))
```

Comment:
`map` might be better as `for/list` here, since it avoids the dangling `points))` line. Could be a good heuristic/test case for Resyntax?

```
(for/list ([x points])
  (list (car x)
        (- (list-ref x 1) min-x)
        (- (list-ref x 2) min-y)))

Reply:
Yeah, I think that's worth doing. It could be similar to the current `for-each` to `for` rule, which only fires when `for-each` uses a multi-line lambda. Can you create an issue in the Resyntax repository for this?

Reply:
jackfirth/resyntax#193